src/test/java/com/googleinterns/zoomtube/servlets/DiscussionServletTest.java

```diff
@@ -180,19 +210,12 @@ public void doGet_returnsCommentsForSpecificLecture() throws ServletException, I
   }
 
   private Entity createTestCommentEntity(int lectureId) {
-    Entity commentEntity = new Entity(CommentUtil.KIND);
-    Key lecture = KeyFactory.createKey(/* kind= */ "Lecture", lectureId);
-    commentEntity.setProperty(CommentUtil.LECTURE, lecture);
-    commentEntity.setProperty(CommentUtil.PARENT, null);
-    commentEntity.setProperty(CommentUtil.TIMESTAMP_MS, 0);
-    // Most properties here are not tested, but are required by the AutoValue class so must be
-    // specified.
-    commentEntity.setProperty(
-        CommentUtil.AUTHOR, new User("untestedAuthor@example.com", "untested.com"));
-    commentEntity.setProperty(CommentUtil.CONTENT, "Untested content");
-    commentEntity.setProperty(CommentUtil.CREATED, new Date(Clock.systemUTC().millis()));
-
-    return commentEntity;
+    Key lectureKey = KeyFactory.createKey(LectureUtil.KIND, lectureId);
+    User author = new User("test@example.com", "example.com");
```

Comment:
Remember for raw arguments you want to add a comment. It is not obvious what "test@example.com" and "example.com" represent.

Please see: (Document ambiguous literal arguments at the call site) https://docs.google.com/document/d/1ZQz9VulpweKEbg28epLO_tZhCboRF1DtWLrVZZlUYBE/edit#

Reply:
Added

Reply:
They don't seem to be added?

Reply:
Still not changed.